src/app.css

```diff
@@ -0,0 +1,17 @@
+body {
+  font-family: 'Comic Sans';
```

Comment:
Should specify fallback, eg:
```suggestion
  font-family: 'Comic Sans, Arial, sans-serif;
```